practice-one/src/styles/components/menu.css

```diff
@@ -0,0 +1,14 @@
+.menu-heading {
+  font-size: var(--font-size-sub-header);
+  font-weight: 700;
+  line-height: 1.4;
```

Comment:
Do we need variable for font weight and line height similar to font size?

Reply:
Yes, that's can be reused many time. So i will make it